app/page-partials/my-dashboard/TeamInfoTabs.tsx

```diff
@@ -240,13 +233,13 @@ function TeamInfoTabs({ alert, currentUser, team, loadTeams }: Props) {
   const [loading, setLoading] = useState(false);
   const [deleteMemberId, setDeleteMemberId] = useState<number>();
   const [modalType, setModalType] = useState('allow');
-  const canDeleteServiceAccount = integrations.length === 0;
   const openModal = () => (window.location.hash = addMemberModalId);
   const inProgressServiceAccount = serviceAccount?.status !== 'applied' && !serviceAccount?.archived;
-  const showDeleteServiceAccountModal = () => (window.location.hash = deleteServiceAccountModalId);
+  const showDeleteServiceAccountModal = () => {
+    window.location.hash = deleteServiceAccountModalId;
+  };
 
   const handleDeleteSeviceAccount = async () => {
-    if (!canDeleteServiceAccount) return;
     await deleteServiceAccount(team.id, serviceAccount?.id);
```

Comment:
Do you know if there was a reason the delete button was conditionally rendered in the handleDeleteServiceAccount function?

Reply:
The previous requirement was to allow deletion only after deleting all the team integrations but the new requirement has removed that need so the delete button is displayed irrespective of number of active integrations on the team